kafka-elastic/mvnw

```diff
@@ -0,0 +1,259 @@
+#!/bin/sh
```

Comment:
Avoid committing the mvnw files, I don't see other articles with these

Reply:
removed